lib/client/bst-alexa.ts

```diff
@@ -68,17 +68,19 @@ export class BSTAlexa {
                        private intentSchemaFile?: string,
                        private sampleUtterancesFile?: string,
                        private applicationID?: string) {
-        if (this.intentSchemaFile === undefined || this.intentSchemaFile === null) {
+        if (!this.intentSchemaFile) {
             this.intentSchemaFile = BSTAlexa.DefaultIntentSchemaLocation;
         }
 
-        if (this.sampleUtterancesFile === undefined || this.sampleUtterancesFile === null) {
+        if (!this.sampleUtterancesFile) {
             this.sampleUtterancesFile =  BSTAlexa.DefaultSampleUtterancesLocation;
         }
 
         this._alexa = new Alexa();
-        if (this.applicationID !== undefined && this.applicationID !== null) {
+        if (this.applicationID) {
```

Comment:
This set the application id in the machine configuration file, if there's no id present we shouldn't save an empty value.